src/templates/ratings/add.html

```diff
@@ -1,6 +1,6 @@
 {% macro review_prompt(slug, isModal, existingReview) %}
```

Comment:
From the AMO days we had a style guide that directed us to use camelCase (which seems common for JS vs underscores for Python) so it made it into Fireplace also.